File: src/DexaWeb.Server/Models/Dexa/AssetStatus.cs
namespace DexaWeb.Server.Models.Dexa;

/// <summary>
/// 에셋 실시간 상태 (asset_status 테이블).
/// DEXA 2.20 Topic 시스템이 주기적으로 갱신.
/// </summary>
public class AssetStatus
{
    public int AssetId { get; set; }
    public int Available { get; set; }
    public DateTime? Time { get; set; }
    public string? Status { get; set; }
    public string? Detail { get; set; }
}

File: src/DexaWeb.Dexa/Models/AssetStatus.fs
namespace DEX.Core.Database.ORM

open System

type AssetStatus() =
    member val AssetId: int = 0 with get, set
    member val Available: int = 0 with get, set
    member val Time: Nullable<DateTime> = Nullable<DateTime>() with get, set
    member val Status: string = null with get, set
